packages/admin-test-utils/lib/fixtures/collectionTypes/address.js

```diff
@@ -401,7 +401,7 @@ const addressCT = {
   isDisplayed: true,
   apiID: 'address',
   kind: 'collectionType',
-  info: { displayName: 'addresse', name: 'address', description: '', label: 'Addresses' },
+  info: { displayName: 'addresses', name: 'address', description: '', label: 'Addresses' },
```

Comment:
I'm kinda worried about this one.

Reply:
It's just a test package, the name doesn't really matter here